src/main/java/org/jabref/gui/groups/GroupSidePane.java

```diff
@@ -20,15 +25,43 @@
 public class GroupSidePane extends SidePaneComponent {
 
     private final JabRefPreferences preferences;
+    private final DialogService dialogService;
+    private final Button intersectionUnionToggle = IconTheme.JabRefIcons.WWW.asButton();
+    private final Tooltip toggleUnion = new Tooltip(Localization.lang("Toogle union"));
```

Comment:
I think there is no need to have these tooltips as instance variables. I would simply inline them.